src/lambda_streams/Numbers.java

```diff
@@ -1,91 +1,306 @@
+        return sum;
+    };
+        System.out.println(lambdaSubtracted.calculate()+" is the sum");
+
+        LambdaSequence2 lambdaMultiplied = () ->{
+        double sum = nums.stream()
+                .mapToDouble(x->x)
+                .reduce(1d,(a,b)-> a*b);
+        return sum;
+    };
+        System.out.println(lambdaMultiplied.figure()+" is the sum");
+
+        LambdaSequence2 lambdaDivided = () ->{
+        double sum = nums.stream()
+                .mapToDouble(x->x)
+                .reduce(1d,(a,b)-> a/b);
+        return sum;
+    };
+        System.out.println(lambdaDivided.figure()+" is the sum");
+
+
+    LambdaInteger lambdaFindMax = () -> {
+        Integer max = nums.stream()
+                .max(Comparator.comparing(Integer::valueOf))
+                .get();
+        return max;
+    };
+        System.out.println(lambdaFindMax.find()+" is the max");
+
+    LambdaInteger lambdaFindMin = () -> {
+        Integer min = nums.stream()
+                .min(Comparator.comparing(Integer::valueOf))
+                .get();
+        return min;
+    };
+        System.out.println(lambdaFindMin.find()+" is the min");
+
+
+    LambdaCompare lambdaCompare = (firstIndex,secondIndex)-> {
+            int firstInt = nums.stream()
+                    .filter(x -> x.equals(nums.get(firstIndex)))
+                    .findFirst().get();
+
+            int secondInt = nums.stream()
+                    .filter(x -> x.equals(nums.get(secondIndex)))
+                    .findFirst().get();
+
+            if (firstInt > secondInt) {
+                return 1;
+            } else if (firstInt < secondInt) {
+                return -1;
+            }
+            return 0;
+    };
+        System.out.println(lambdaCompare.comp(2,5));
+        System.out.println(lambdaCompare.comp(5,2));
+        System.out.println(lambdaCompare.comp(0,13));
+
+
+
+
+    LambdaNewElement lambdaAppend = (newElement) -> {
+        Stream<Integer> newStream = Stream.concat(nums.stream(), Stream.of(newElement));
+                int newInt = newStream
+                    .skip(nums.stream().count()).findFirst().get();
+                return newInt;
+        };
+        System.out.println(lambdaAppend.append(77));
+
+
 
-    static int added() {
-        //add all the elements in the list.  return the sum.  
-        return 0;
-    }
 
-    static int subtracted() {
-        //subtract all the elements in the list. return the remainder.
-        return 0;
-    }
 
-    static int multipled() {
-        //multiply all the elements in the list. and return the product.
-        return 0;
-    }
 
-    static int divided() {
-        //multiply all the elements in the list. and return the product.
-        return 0;
-    }
 
-    static int findMax() {
-         //return the maximum value in the list.
-        return 0;
     }
 
-    static int findMin() {
-        //return the minimum value in the list.
-        return 0;
+}
+    interface LambdaBoolean{
```

Comment:
Each .java file is expected to have only one class (or interface) at the top-level.  Including more than one adds risk that a developer may struggle to find the code for which they are looking, and if they aren't careful, could end up blending code from one class or interface into another.